apps/system/js/app_text_selection_dialog.js

```diff
@@ -67,6 +61,22 @@
 
   AppTextSelectionDialog.prototype.ELEMENT_PREFIX = 'textselection-dialog-';
 
+  AppTextSelectionDialog.prototype.onObservePrefChanged =
+    function tsd_onObservePrefChanged(value) {
+      if (value) {
+        this.start();
+      } else {
+        this.stop();
+      }
+    }.bind(this);
```

Comment:
Thanks for the feedbacks. I've written a new version and made sure that the feature is still working on my local device.